.github/workflows/check-symbol.yml

```diff
@@ -0,0 +1,103 @@
+name: Check Symbols
+
+on:
+  workflow_run:
+    workflows: ["Build"]
+    types:
+      - requested
+      - completed
+
+jobs:
+  pending:
+    if: |
+      github.event.action == 'requested' &&
+      github.event.workflow_run.event == 'pull_request'
```

Comment:
Why do we need this job?

Reply:
This is a job to display the "Check Symbols" checker status as below when build starts.
![image](https://user-images.githubusercontent.com/1029205/135229002-a868fb98-e6a3-4cbe-9efa-7d9d9346b0f1.png)

Reply:
Hmm. But, why? Those three steps add extra complexity to the workflow file and the information doesn't seem very useful. Won't `check-symbol.py` exit with non-zero exit code if anything goes wrong?

Also, how does separating the Check Symbols workflow from the Build workflow improve security? Is it different from adding a job that depends on the build job in the Build workflow?

Reply:
1. I agree, `pending` job is not necessary. But without this job, the "Check Symbol" checker will be shown in the PR status only after the build is over.
2. To check the symbol, the workflow have to access the allowlist in private repo with the secret `TIZENAPI_TOKEN'. However, for security reasons, these secret keys are not accessible in the forked pull request event workflows. 
- https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
- https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_run

Reply:
> But without this job, the "Check Symbol" checker will be shown in the PR status only after the build is over.

That's okay. I don't see any problem with it.

> However, for security reasons, these secret keys are not accessible in the forked pull request event workflows.

Okay. Now I understand that it's one of GitHub's limitations. I have a slightly different idea but I'll talk to you offline later.